components/sidebar/sidebar_next_steps/close_next_steps_modal.tsx

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {FormattedMessage} from 'react-intl';
+
+import FormattedMarkdownMessage from 'components/formatted_markdown_message.jsx';
+import GenericModal from 'components/generic_modal';
+import closeNextStepsArrow from 'images/close_next_steps_arrow.svg';
+
+import './close_next_steps_modal.scss';
+
+type Props = {
+    screenTitle: string;
+    onConfirm: () => void;
+    onCancel: () => void;
+}
+
+export default function CloseNextStepsModal(props: Props) {
```

Comment:
Can we rename this to "RemoveNextStepsModal" ? Initially it was a bit confusing because it gave me the idea that it's a 'Close button' component.